stoic.go

```diff
@@ -31,5 +33,15 @@ func main() {
 		return //We're done here
 	}
 
+	if *editFlag != "" {
```

Comment:
I would propose instead of multiple `if`'s to use `if elsif`.
```
if *aboutFlag != "" {
} elsif *editFlag != "" {
} else {
  _ = ctx.OpenInEditor(stoic.NewEntry(ctx, time.Now()))
}
```

This way, we're sure to execute only one command.